docs/framework/docker/aspnetmvc.md

```diff
@@ -50,33 +45,31 @@ The finished application is located in the
 
 ## Prerequisites
 
-At a minimum, your development machine must be running
+The development machine must be running
 [Windows 10 Anniversary Update](https://www.microsoft.com/en-us/software-download/windows10/)
 or [Windows Server 2016](https://www.microsoft.com/en-us/cloud-platform/windows-server). 
 
-Before starting, you need to install [Docker for Windows](https://docs.docker.com/docker-for-windows/),
+[Docker for Windows](https://docs.docker.com/docker-for-windows/) must be installed,
```

Comment:
The part about the Beta channel seems outdated. Latest sable release seems to have added that: https://docs.docker.com/docker-for-windows/release-notes/#/docker-for-windows-1130-2017-01-19-stable